fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -303,6 +311,91 @@ private fun SettingsCard(
     }
 }
 
+@Composable
+private fun ReviewQualityInfoCard(
+    onLearnMoreClick: () -> Unit,
```

Comment:
think we can have a modifier param here and the parent/caller can set the max width for it

Reply:
When it's a child composable that's only ever used in one manner, I'm honestly never completely sure if it's worth adding, out of caution of an accidental modifier regression, but it might make more sense since it _is_ a child composable to give the parent caller the responsibility of giving it the correct sizing modifiers, so I'll add the parameter here and to `ReviewGradingScaleInfo`